src/Spryker/Zed/AppKernel/Business/Writer/ConfigWriter.php

```diff
@@ -54,6 +55,8 @@ public function saveConfig(AppConfigTransfer $appConfigTransfer): AppConfigRespo
     {
         try {
             $appConfigTransfer = $this->getTransactionHandler()->handleTransaction(function () use ($appConfigTransfer) {
+                $appConfigTransfer = $this->mergeWithExistingConfiguration($appConfigTransfer);
```

Comment:
it just sets the status
```suggestion
                $appConfigTransfer = $this->updateStatus($appConfigTransfer);
```

Reply:
The method name is open for anything else when we need it. Do you want to change anyway?

Reply:
yes, YAGNI

+ it's just protected, you can always change it.